src/core/resources/assets/userTestnetNativeAsset.ts

```diff
@@ -0,0 +1,122 @@
+import { useQuery } from '@tanstack/react-query';
+import { getProvider } from '@wagmi/core';
+import { Address } from 'wagmi';
+
+import {
+  QueryConfig,
+  QueryFunctionArgs,
+  QueryFunctionResult,
+  createQueryKey,
+} from '~/core/react-query';
+import { SupportedCurrencyKey } from '~/core/references';
+import { ParsedUserAsset } from '~/core/types/assets';
+import { ChainId, ChainName, ChainNameDisplay } from '~/core/types/chains';
+import { fetchAssetBalanceViaProvider } from '~/core/utils/assets';
+import { getChain } from '~/core/utils/chains';
+
+const USER_ASSETS_REFETCH_INTERVAL = 60000;
+
+const getNativeAssetMock = ({ chainId }: { chainId: ChainId }) => {
+  const chain = getChain({ chainId });
+  const nativeAssetMock = {
+    address: 'eth',
```

Comment:
I think you can use `NATIVE_ASSETS_PER_CHAIN` from `'~/core/references'` to help here

Reply:
using and updated NATIVE_ASSETS_PER_CHAIN with testnets